README.md

```diff
@@ -99,25 +99,23 @@ instead of being passed into the constructor.
 ### Example Usage
 
 ```js
-var Filter = require('broccoli-persistent-filter');
-
-Awk.prototype = Object.create(Filter.prototype);
-Awk.prototype.constructor = Awk;
-function Awk(inputNode, search, replace, options) {
-  options = options || {};
-  Filter.call(this, inputNode, {
-    annotation: options.annotation
-  });
-  this.search = search;
-  this.replace = replace;
+const Filter = require('broccoli-persistent-filter');
+
+class Awk extends Filter {
+  constructor(inputNode, search, replace, options) {
+    options = options || {};
```

Comment:
is there a reason to do this, rather than `options = {}` in the constructor signature?